services/blue/src/commands/Response.cs

```diff
@@ -7,13 +7,12 @@ public class Response<TValue>
     {
         public bool IsSuccessful => Exception is null;
 
-        [MaybeNull]
-        [AllowNull]
+        [AllowNull, MaybeNull]
```

Comment:
`AllowNull` is for parameters.
```suggestion
        [MaybeNull]
```

Reply:
`[AllowNull]` is enabling to set the property to `null` via the constructor accepting a value: `null` values are allowed since this type can be used as a generic response wrapper.
Without it we get: `[CS8601] Possible null reference assignment` in the constructor.

Reply:
I find that really confusing, because it's a getter only property. If we leave out the `AllowNull` in the constructor, it also works and it would be less confusing. If I use the constructor with the value, I won't pass `null` most of time, so we can just use `MayBeNull` on the property.